django-stubs/contrib/admindocs/middleware.pyi

```diff
@@ -10,6 +10,6 @@ class XViewMiddleware(MiddlewareMixin):
         self,
         request: HttpRequest,
         view_func: Callable[..., HttpResponseBase],
```

Comment:
Just from the structure it looks like it can be a `ParamSpec` thing, can't it?

Reply:
Never mind, it is not `*` and `**`

Reply:
I don't think so... `process_view()` methods don't call `view_func`. Their return values can come only from the middleware.

https://docs.djangoproject.com/en/5.1/topics/http/middleware/#process-view